examples/connext_dds/lbediscovery_xml_app_creation/c++11/README.md

```diff
@@ -0,0 +1,142 @@
+# Example Code: LBED and XML-Based Application Creation
+
+## Building the Example :wrench
+
+To build this example, first run CMake to generate the corresponding build
+files. We recommend you use a separate directory to store all the generated
+files (e.g., ./build).
+
+```sh
+mkdir build
+cd build
+cmake ..
+```
+
+Once you have run CMake, you will find a number of new files in your build
+directory (the list of generated files will depend on the specific CMake
+Generator). To build the example, run CMake as follows:
+
+```sh
+cmake --build .
+```
+
+**Note**: if you are using a multi-configuration generator, such as Visual
+Studio solutions, you can specify the configuration mode to build as follows:
+
+```sh
+cmake --build . --config Release|Debug
+```
+
+Alternatively, you can directly use the generated infrastructure (e.g.,
+Makefiles or Visual Studio Solutions) to build the example. If you generated
+Makefiles in the configuration process, run make to build the example. Likewise,
+if you generated a Visual Studio solution, open the solution and follow the
+regular build process.
+
+## Running the Example
+
+Open two separate command prompt windows for the publisher and subscriber. Run
+the following commands from the build directory (this is necessary to ensure
+the application loads the QoS defined in *USER_QOS_PROFILES.xml*):
+
+On *Windows* systems run:
+
+```sh
+ShapeType_publisher.exe  -d <domain_id> -s <samples_to_send> -v <verbosity 0-3>
+ShapeType_subscriber.exe -d <domain_id> -s <samples_to_receive> -v <verbosity 0-3>
+```
+
+On *UNIX* systems:
+
+```sh
+./ShapeType_publisher  -d <domain_id> -s <samples_to_send> -v <verbosity 0-3>
+./ShapeType_subscriber -d <domain_id> -s <samples_to_receive> -v <verbosity 0-3>
+```
+
+The applications accept up to three arguments:
+
+1. The `<domain_id>`. Both applications must use the same domain ID in order to
```

Comment:
```suggestion
1.  The `<domain_id>`. Both applications must use the same domain ID in order to
```
Can you add a space here and before all other numbered and unnumbered lists? (63, 66. 77, 82, 87, 126 29 and 131)